torax/geometry_loader.py

```diff
@@ -58,6 +59,11 @@ def _load_fbt_data(file_path: str) -> dict[str, np.ndarray]:
   """Loads the data from a FBT-LY file into a dictionary."""
   return scipy.io.loadmat(file_path, squeeze_me=True)
 
+from eqdsk import EQDSKInterface
+def _load_eqdsk_data(file_path: str) -> dict[str, np.ndarray]:
+    eq = EQDSKInterface.from_file(file_path, no_cocos=True) # should probably handle COCOS shenanigans...
+    return eq.__dict__
```

Comment:
Use dict(eq) but also maybe rename to something like loaded_data or similar as eq is generally used for equals

Reply:
dict(eq) did not work, therefore I have kept it as eq.__dict, except renaming as suggested.